editor.planx.uk/src/@planx/components/shared/Preview/SimpleExpand.tsx

```diff
@@ -16,16 +26,16 @@ const SimpleExpand = ({ children, buttonText }: any) => {
   const classes = useClasses();
   return (
     <>
-      <Box className={classes.root}>
-        <Button
-          size="large"
-          fullWidth
-          color="inherit"
-          onClick={() => setShow(!show)}
-        >
-          {show ? buttonText.closed : buttonText.open}
-        </Button>
-      </Box>
+      <Button
+        size="large"
+        disableRipple
+        className={classes.button}
+        onClick={() => setShow(!show)}
+        // Needs aria-controls, aria-expanded
```

Comment:
thanks for leaving these notes - happy to pick them up on the main branch after styles merge in :slightly_smiling_face: